doc/vitup/mock.md

```diff
@@ -37,6 +37,53 @@ curl --location --request GET 'http://{mock_address}/api/health'
 
 ```
 curl --location --request POST 'http://{mock_address}/api/control/command/fund/id/{new_fund_id}'
+
+##### Add new fund
```

Comment:
I'm not actually sure if we need this in the mock, but it doesn't hurt anyway.

I guess it could be used to test that the changes are actually shown in the app, but since the app doesn't cache anything it's more or less granted.